### PR TITLE
Add docstrings for py2ju() and retrieve()

### LINKED ISSUE #16: Add docstrings for the two functions
It would be nice to have some docstrings.

### PR DESCRIPTION
**Pull request by LakshyaKhatri**
Fixes #16

### CODE REVIEW

File: src/CDSAPI.jl [hunk 1]
```diff
@@ -7,7 +7,16 @@ using Base64
 export
     py2ju
 
-function retrieve(name, params, filename)
+"""
+    retrieve(name::AbstractString,
+        params::Dict,
```
Comment (juliohm) [inline src/CDSAPI.jl:12]: Please make sure you are not inserting real TAB in your text editor. Do you mind aligning the arguments and removing the type annotation?

Comment (LakshyaKhatri) [reply src/CDSAPI.jl:12]: You mean I should configure the editor to insert 4 spaces instead of a TAB?

Comment (juliohm) [reply src/CDSAPI.jl:12]: Yes, TABs are discouraged in general.

Comment (LakshyaKhatri) [reply src/CDSAPI.jl:12]: and should I remove type annotations from both docstrings and fuction defination?

Comment (LakshyaKhatri) [reply src/CDSAPI.jl]: A bit late but done. I was in a session :grimacing:

**Comment by juliohm** [comment]
Awesome, thanks for the improvements! :)

**Pull request merged**